src/components/ExternalButtonLink.tsx

```diff
@@ -0,0 +1,20 @@
+import { Button, ButtonProps } from "@chakra-ui/react"
+import { ExternalLinkHref } from "./Navbar/types"
+import { FC } from "react"
+
+const ExternalButtonLink: FC<ButtonProps & { href: ExternalLinkHref }> = ({
```

Comment:
this should resolve your comment on the target PR: https://github.com/threshold-network/splash-page/pull/10/files#r822760408